### ISSUE TITLE
imgHasAlt triggered even if aria-label used

### ISSUE THREAD

**Issue by robfentress** [ucfopen/UDOIT#97]
The imgHasAlt rule is triggered when an `<img>` tag doesn't have an `alt` attribute set, even if the `<img>` has alternative text using `aria-label`.

![screen shot 2016-02-24 at 3 55 18 pm](https://cloud.githubusercontent.com/assets/9356499/13300726/810b5060-db0f-11e5-9509-68935b3888c9.png)
